projectDocs/Design Implementation

```diff
@@ -0,0 +1,35 @@
+# Design (Subject to updates)
+- All uses base view from submodule
+- Student/Staff submit new issue
+  - login is optional, login with D&E info else anonymous
+    - If anonymous then ask for contact details?
+  - Restful API to database 
+    - Database will have: 
+      - pk
+      - subject
+      - Issue content
+      - contactName
+      - dateSubmitted
+      - issue status (reference codes)  
+      
+        |  statusCode  |      StatusLabel      |  
+        |     -----    |         -----         |  
+        |      10      |  "Request Received"   |  
+        |      20      |  "Problem Confirmed"  |  
+        |      30      |  "Fix Defined"        |  
+        |      40      |  "Fix Implemented"    |  
+        |      50      |  "Fix Confirmed"      |  
+      - Github issue number
+      - Github repository (probably don't need)
+- Should be able to view/add comments to it.  CAN NOT update original issue.
+    - Adding or viewing old Issues means goign back into the repository, should only students or whoever has a legitimate login be able to view their old Issues?  Just asking the database for issues associated to a certain email or contact credential seems like poor usage/confidentiality of a persons credentials.
```

Comment:
Yes, should probably allow only non-anonymous users to view submitted information in database. For now, that means D&E users. Anonymous users who have submitted the form as a contact/information request will be contacted via email, so they will not need to review the status of their submission. Later, we will add a kind of authentication for non-D&E users so that they can see their database contents securely.